### DISCUSSION TITLE
Combine unloading when trailer arrives at field

### DISCUSSION THREAD

**Discussion by Redster999** [Courseplay/Courseplay_FS22#2863] [Ideas]
I asked for this earlier, and Tensuko made a note of it.
I dunno if he or they forgot but heres a reminder.

I love the feature that the combine unloads to trailers parked at fields. Usually works fine. But once the trailer leaves to unload somewhere a bit far away, it might takes a while for the trailer to return. If the combine fills up before the trailer arrives, the combine stands still and just wait to be emptied by a trailer that is not coming. When the trailer show up, its just waiting.

There is a feature that an overloader-trailer stands in the field is waiting to see when a trailer arrives, it can go and unload.
How about adding this overloader-looks-for-trailer feature to the combine, so it will continue to work again on its own?

**Comment by Redster999** [comment]
The new showel mode also does work for this. The showel-tractor waits for the trailer to come back before it continues. And it works fine. I hope the idea is solvable ;)

**Reply by Tensuko** [reply]
Yea that is the recheck for trailers, I have not rly forget about this but I know that @schwiti6190 is working on a refactoring for some of this stuff and I hope I remember to ask him for this. :)

For auger wagons it should exist already too.

**Reply by Redster999** [reply]
Thank you Tensuko!

**Reply by Torqueitup** [reply]
I was also wondering what would be the viability of being able to drop off a 'skip' on the side of the field for the Harvester to unload into while the tractor goes off and does work elsewhere?  I use the iRunner skips/trailers which are dirt cheap to buy compared to comparable trailers of similar capacity and to be able to drop these off at various fields for Harvesters to unload to would be interesting.

**Comment by Redster999** [comment]
Hello, Just a friendly reminder. Has this been considered?

**Reply by Tensuko** [reply]
Not yet, you will see that on the changelog when it is done!
